components/item/MediaActionBar.stories.js

```diff
@@ -31,12 +51,14 @@ storiesOf('Record / Media thumbnail grid', module)
         selected: 'http://www.mimo-db.eu/media/GNM/IMAGE/MIR1097_1279787057222_2.jpg'
       };
     },
-    template: ` <b-container
-      class="mt-3"
-      >
-        <MediaThumbnailGrid
-          :media="media"
-          :selected="selected"
-        />
-      </b-container>`
+    template: `
+      <b-container class="mt-3">
+        <div class="card p-3 mb-3 mt-3 bg-grey">
+          <MediaActionBar
```

Comment:
Could you add a download button too, as well as a rights statement and an example of a "view at" to match its current state on item pages?